### DISCUSSION TITLE
Add the UserConfig type to the czg library as well

### DISCUSSION THREAD

**Discussion by curlykay9527** [Zhengqbbb/cz-git#135] [Q&A] [answered]
I'm only using `czg` in my project now and would like to have a method like `cz-git` to get the configuration type definition
like:
```
/** @type {import('czg').UserConfig} */
module.exports = {
```

**Comment by Zhengqbbb** [comment]
Due to the size, the current version `js configure file` does not have it. I am currently open to this discussion and am not sure if it will be added in my next version. 

Tip If you use json configuration, you can use `JSON schema`

```json
{
  "$schema": "https://cdn.jsdelivr.net/gh/Zhengqbbb/cz-git@1.6.1/docs/public/schema/cz-git.json"
}
```

**Comment by curlykay9527** [comment]
Unfortunately, I don't use json configuration and look forward to the next version.
by the way this is a great library!

**Reply by Zhengqbbb** [reply]
I will notify you if `czg` release , Thx you for your support. 🤗

**Comment by Zhengqbbb** [comment, marked as answer]
The latest version 🌟(v1.7.0)🌟 should be resolve this issue request , please have a try. 💪
👀 Let me know if there are still any issues. 🎉

https://cz-git.qbb.sh/cli/#quick-start

**Reply by curlykay9527** [reply]
Great DX, works perfectly in my project.
